cards/production/13TeV/higgs/ggh01_M125_Toa01a01_M20_Tomumubb/ggh01_M125_Toa01a01_M20_Tomumubb_run_card.dat

```diff
@@ -0,0 +1,282 @@
+ -1  = htjmax ! maximum jet HT=Sum(jet pt)
+ 0   = ihtmin  !inclusive Ht for all partons (including b)
+ -1  = ihtmax  !inclusive Ht for all partons (including b)
+ 0   = ht2min ! minimum Ht for the two leading jets
+ 0   = ht3min ! minimum Ht for the three leading jets
+ 0   = ht4min ! minimum Ht for the four leading jets
+ -1  = ht2max ! maximum Ht for the two leading jets
+ -1  = ht3max ! maximum Ht for the three leading jets
+ -1  = ht4max ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+   0 = ptgmin ! Min photon transverse momentum
+ 0.4 = R0gamma ! Radius of isolation code
+ 1.0 = xn ! n parameter of eq.(3.4) in hep-ph/9801442
+ 1.0 = epsgamma ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ .true. = isoEM ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+ 0   = xetamin ! minimum rapidity for two jets in the WBF case  
+ 0   = deltaeta ! minimum rapidity for two jets in the WBF case 
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+ -1    =  ktdurham        
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 0   = xqcut   ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+#   F  = use_syst      ! Enable systematics studies
+   F  = cut_decays      ! Enable systematics studies
```

Comment:
Hi,

This is strange since in my local files I don't have the commented line!!!!

No clue what went wrong.

Cheers,
Nadjieh



Sent from Samsung tablet.


-------- Original message --------
From: perrozzi <notifications@github.com>
Date: 21/05/2016 12:58 (GMT+01:00)
To: cms-sw/genproductions <genproductions@noreply.github.com>
Cc: Abideh Jafari <Abideh.Jafari@cern.ch>, Author <author@noreply.github.com>
Subject: Re: [cms-sw/genproductions] 19 may (#875)


In cards/production/13TeV/higgs/ggh01_M125_Toa01a01_M20_Tomumubb/ggh01_M125_Toa01a01_M20_Tomumubb_run_card.dat<https://github.com/cms-sw/genproductions/pull/875#discussion_r64132718>:

> +#*********************************************************************
> + 4 = maxjetflavor    ! Maximum jet pdg code
> +#*********************************************************************
> +# Jet measure cuts                                                   *
> +#*********************************************************************
> + 0   = xqcut   ! minimum kt jet measure between partons
> +#*********************************************************************
> +#
> +#*********************************************************************
> +# Store info for systematics studies                                 *
> +# WARNING: If use_syst is T, matched Pythia output is                *
> +#          meaningful ONLY if plotted taking matchscale              *
> +#          reweighting into account!                                 *
> +#*********************************************************************
> +#   F  = use_syst      ! Enable systematics studies
> +   F  = cut_decays      ! Enable systematics studies


need to fix here as well

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/genproductions/pull/875/files/b12f3bed7062aca9eb0afe7488f6cb46203b487f#r64132718>